src/common/commonutils/CommandUtils.c

```diff
@@ -29,270 +31,200 @@ static int NormalizeStatus(int status)
     return newStatus;
 }
 
-static int SystemCommand(void* context, const char* command, int timeoutSeconds, CommandCallback callback, OsConfigLogHandle log)
-{
-    const int callbackIntervalSeconds = 5; //seconds
-    const int defaultCommandTimeout = 60; //seconds
```

Comment:
What's the callback frequency with this new version of the command? Do we keep the previous 5 seconds?